### DISCUSSION TITLE
How to get raw markdown

### DISCUSSION THREAD

**Discussion by gempain** [outline/outline#2233] [Self Hosting] [answered]
Hi !
Is it possible to find the raw markdown ? I need it to copy/paste some content from Outline to Slack.
Thanks a lot !

**Comment by tommoor** [comment, marked as answer]
Click on the "..." menu on a document and select "Download" at the bottom, you'll get a pure Markdown file!

**Reply by gempain** [reply]
This is awesome, thanks !
